### DISCUSSION TITLE
Decoding "stange" broadcast messages for Ariston Ebus

### DISCUSSION THREAD

**Discussion by wrongisthenewright** [john30/ebusd#880] [Q&A]
Hi, 
before opening an issue I'll ask here to be sure I'm not forgetting something.
I have a working ebusd config file reading many parameters from my Ariston system (hybrid HP+boiler), I'm decoding many broadcast messages but one specific telegram didn't get decoded by ebusd.

The message is:
`13fe201008c64bbe00c94b0200`

In Ariston "jargon" (Bridgenet protocol over Ebus link layer) meaning is 
`13 fe 2010 08 c64b be00 c94b 0000` where 
c64b and c94b are parameters and be00 and 0000 are, at least what I think, temp values.

I try to grab the messages with this line:
r,test,c64b_c94b_unk_13_fe,UNKN,13,fe,2010,c64b,c64b_SIN,m,SIN,10,°C,,,m,IGN:2,,,,c84b_SIN,m,SIN,10,°C

but Ebusd keeps failing with:

2023-04-10 17:24:56.073 [update info] received BC cmd: 13fe201008c64bbe00c94b0000
2023-04-10 17:24:56.073 [update notice] received unknown BC cmd: 13fe201008c64bbe00c94b0000

The only explanation I have is that the message is malformed as there's no `/00` ACK at the end of the message.
Any suggestion on how to set the CSV?

TY

**Comment by DaeMonSxy** [comment]
I seem to be in a similar shoe.

Any success around Ariston csv? ( Iam having Arison Genus One 30)
Thanks

```
[update notice] received unknown BC cmd: 37fe20100cc04b0100091900000b190000
2025-02-08 08:58:20.939 [update notice] received unknown BC cmd: 37fe20101001910000029100000391000004910000
2025-02-08 08:58:21.053 [update notice] received unknown BC cmd: 37fe2010080591000006910000
2025-02-08 08:58:21.172 [update notice] received unknown BC cmd: 37fe20700736080802061901

```

**Comment by wrongisthenewright** [comment]
HI, 
I have the same boiler albeit in an hybrid system that has also an HP.
My file has evolved since I posted this and solved many of the problems I had (due mostly to my not understanding on how ebusd managed broadcasted messages).

My file is here:

https://github.com/wrongisthenewright/ebusd-configuration-ariston-bridgenet

Someone reported that in a boiler only system configuration doesn't work very well, someone has then used Ysard's version of the file (he has a boiler only configuration)

https://github.com/ysard/ebusd_configuration_chaffoteaux_bridgenet

Make some test with both and see what suit you better...
